themes/Default/Dark.qss

```diff
@@ -1052,15 +1054,15 @@ TalkingUI > * {
   background-color: #191919;
 }
 
-TalkingUI [selected=false] {
+TalkingUI [selected="false"] {
```

Comment:
Did you endure that the TalkingUI still allows for selectable users and channels with this change applied. Off the top of my head I'm not entirely sure whether the code might expect a Boolean value and with this change might end up getting fed a string :thinking:

What's the rationale for this part of the change anyway?

Reply:
@Krzmbrzl No, but I can test that later. The Dark.qss and Lite.qss files are compiled by sass and I did not modify them by hand (as it should be done according to the README in themes). It simply seems like newer versions of sass make sure that these parameters are strings in the qss files...

Reply:
Yeah, I figured. We just have to make sure thst this doesn't break anything :)

Reply:
@Krzmbrzl 

![screen5](https://user-images.githubusercontent.com/1348773/183051661-d39efbb6-7efd-4b47-8226-359d9b58ff89.png)

Seems to work, right?

Reply:
Yes, that's looking good :+1: